pkg/tso/keyspace_group_manager.go

```diff
@@ -617,10 +613,20 @@ func (kgm *KeyspaceGroupManager) updateKeyspaceGroup(group *endpoint.KeyspaceGro
 
 // updateKeyspaceGroupMembership updates the keyspace lookup table for the given keyspace group.
 func (kgm *KeyspaceGroupManager) updateKeyspaceGroupMembership(
-	oldGroup, newGroup *endpoint.KeyspaceGroup,
+	oldGroup, newGroup *endpoint.KeyspaceGroup, updateWithLock bool,
```

Comment:
maybe split it to `updateKeyspaceGroupMembership` and `updateKeyspaceGroupMembershipWithLock`, which will be more easier to read